test/test-project.js

```diff
@@ -53,9 +53,39 @@ test('package info gathering', function(t) {
   self.gather(function(err, project) {
     t.ifErr(err, 'should not error out');
     t.same(self, project);
+    t.equal(project.get('license'), 'MIT');
```

Comment:
You've written "MIT" in a few places, as well as "1.0.0". Might want to throw them into a variable and just reuse.